tgui/packages/tgui/interfaces/MixingDesk.js

```diff
@@ -0,0 +1,108 @@
+import { useBackend, useSharedState } from '../backend';
+import { Button, LabeledList, Section, Divider, Modal, Input } from '../components';
+import { Window } from '../layouts';
+
+export const MixingDesk = (props, context) => {
+  const { act, data } = useBackend(context);
+  const {
+    voices,
+    selected_voice,
+    say_popup,
+  } = data;
+  const [message, setMessage] = useSharedState(context, "message", null);
+
+  const sayPopup = () => (
+    <Modal>
+      Choose something to say as {
+        selected_voice > 0 && selected_voice <= voices.length
+          ? voices[selected_voice - 1].name
+          : "yourself"
+      }:
+      <br />
+      <Input
+        autoFocus
+        selfClear
+        width={20}
+        value={message}
+        onEnter={(_, msg) => {
+          window.focus();
```

Comment:
This is necessary. If you don't move focus away from the Input it will lead to the Dream Seeker window regaining focus once the Input is removed.